src/bindgen/ir/structure.rs

```diff
@@ -549,6 +549,40 @@ impl Source for Struct {
                 self.emit_bitflags_binop('^', &other, out);
             }
 
+            if config.structure.derive_ostream(&self.annotations) {
+                if !wrote_start_newline {
+                    wrote_start_newline = true;
+                    out.new_line();
+                }
+
+                out.new_line();
+                let stream = config
+                    .function
+                    .rename_args
+                    .apply("stream", IdentifierType::FunctionArg);
+                let instance = config
+                    .function
+                    .rename_args
+                    .apply("instance", IdentifierType::FunctionArg);
+                write!(
+                    out,
+                    "friend std::ostream& operator<<(std::ostream& {}, const {}& {})",
```

Comment:
I assume this doesn't need to be inline because it's in the class definition right?

Reply:
Yep, I can add a comment about that.